sdk/vision/imageAnalysis/samples/typescript/analyzeImageFromLocalFile.ts

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as fs from 'fs';
+import createImageAnalysisClient, { ImageAnalysisClient, ImageAnalysisResultOutput } from '@azure/imageAnalysis';
+import { AzureKeyCredential } from '@azure/core-auth';
+// Load the .env file if it exists
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const endpoint: string = process.env['VISION_ENDPOINT'] || '<your_endpoint>';
+const key: string = process.env['VISION_KEY'] || '<your_key>';
+const credential = new AzureKeyCredential(key);
+
+const client: ImageAnalysisClient = createImageAnalysisClient(endpoint, credential);
+
+const features: string[] = [
+  'Caption',
+  'DenseCaptions',
+  'Objects',
+  'People',
+  'Read',
+  'SmartCrops',
+  'Tags'
+];
+const imagePath: string = '../sample.jpg';
+
+async function analyzeImageFromFile(): Promise<void> {
+  const imageBuffer: Buffer = fs.readFileSync(imagePath);
+
+  const result = await client.path('/imageanalysis:analyze').post({
+    body: imageBuffer,
+    queryParameters: {
+      features: features,
+      'smartCrops-aspect-ratios': [0.9, 1.33]
+    },
+    contentType: 'application/octet-stream'
+  });
+
+  const iaResult: ImageAnalysisResultOutput = result.body as ImageAnalysisResultOutput;
+
+  console.log(`Model Version: ${iaResult.modelVersion}`);
+  console.log(`Image Metadata: ${JSON.stringify(iaResult.metadata)}`);
+
+  if (iaResult.captionResult) console.log(`Caption: ${iaResult.captionResult.text} (confidence: ${iaResult.captionResult.confidence})`);
```

Comment:
It is more idiomatic for JS to have the if blocks wrapped in {} even when it is a single line

```ts
if (iaResult.captionResult){ 
   console.log(`Caption: ${iaResult.captionResult.text} (confidence: ${iaResult.captionResult.confidence})`);
}
```